build

```diff
@@ -15,7 +15,9 @@ cd -P -- "$(dirname -- "${BASH_SOURCE[0]}")"
 PKG_VSN="${PKG_VSN:-$(./pkg-vsn.sh)}"
 export PKG_VSN
 
-SYSTEM="$(./scripts/get-distro.sh)"
+if [ "$(uname -s)" != 'Win' ]; then
+    SYSTEM="$(./scripts/get-distro.sh)"
+fi
```

Comment:
else windows？

Reply:
> else windows？

`get-distro.sh` script will cause errors on windows. I am not sure how to generate system variable on windows, so I want to skip it, but it seems that `uname -s` does not return to `Win` on windows?